viewing_party/party.py

```diff
@@ -1,23 +1,142 @@
 # ------------- WAVE 1 --------------------
-
 def create_movie(title, genre, rating):
-    pass
+    if None in (title, genre, rating):
+        return None
+
+    return {"title": title, "genre": genre, "rating": rating}
+
+
+def add_to_watched(user_data, movie):
+    copy_user_data = dict(user_data)
+    copy_user_data["watched"].append(movie)
+
+    return copy_user_data

```

Comment:
We actually do want to change `user_data` directly. The shallow copy here isn't doing much for us here.

```suggestion
    user_data["watched"].append(movie)
    return user_data

```